src/xunit.performance.api/xunit.performance.api.csproj

```diff
@@ -1,36 +1,41 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup Label="Configuration">
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+  </PropertyGroup>
   <PropertyGroup>
     <VersionPrefix>1.0.0-alpha</VersionPrefix>
-    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard1.5</TargetFrameworks>
```

Comment:
From [https://docs.microsoft.com/en-us/dotnet/articles/standard/library](https://docs.microsoft.com/en-us/dotnet/articles/standard/library),
.netstandard1.5 support .NET Framework 4.6.2

---
In reply to: [100389083](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100389083) [](ancestors = 100389083)